registry/test/text-color.test.ts

```
@@ -14,30 +14,58 @@ import {
+    // Select again and clear color using "Default"
     await selectText(editor, 7, 11, 'some')
-    const defaultBtn = page.getByRole('button', { name: 'default' })
+    const defaultBtn = page.getByRole('button', { name: 'Text: Default' })
     await expect.element(defaultBtn).toBeVisible()
     await defaultBtn.click()
 
     // Expect no colored span wraps the word "some"
     await expectLocatorToNotExist(someSpan)
   })
+
+  it('change and clear background color via inline menu', async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)